common/src/main/java/com/microsoft/identity/common/internal/servertelemetry/Schema.java

```diff
@@ -0,0 +1,169 @@
+        public static final String MRRT_STATUS = TelemetryEventStrings.Key.MRRT_STATUS;
+    }
+
+    public static final class Value {
+        public static final String SCHEMA_VERSION = "1";
+        public static final String TRUE = "1";
+        public static final String FALSE = "0";
+        public static final String EMPTY = "";
+    }
+
+    /**
+     * This array defines the common schema for current request.
+     * NOTE: These fields must always be listed in the correct order in this array.
+     * Failure do so will break the schema.
+     */
+    private static final String[] currentRequestCommonFields = new String[]{
+            Key.API_ID,
+            Key.FORCE_REFRESH
+    };
+
+    /**
+     * This array defines the platform schema for current request
+     * NOTE: These fields must always be listed in the correct order in this array.
+     * Failure do so will break the schema.
+     */
+    private static final String[] currentRequestPlatformFields = new String[]{
+            Key.ACCOUNT_STATUS,
+            Key.ID_TOKEN_STATUS,
+            Key.AT_STATUS,
+            Key.RT_STATUS,
+            Key.FRT_STATUS,
+            Key.MRRT_STATUS
+    };
+
+    /**
+     * This array defines the common schema for last request
+     * NOTE: These fields must always be listed in the correct order in this array.
+     * Failure do so will break the schema.
+     */
+    private static final String[] lastRequestCommonFields = new String[]{
+            Key.API_ID,
+            Key.CORRELATION_ID,
+            Key.ERROR_CODE
+    };
+
+    /**
+     * This array defines the platform schema for last request
+     * NOTE: These fields must always be listed in the correct order in this array.
+     * Failure do so will break the schema.
+     */
+    private static final String[] lastRequestPlatformFields = new String[]{
+
+    };
+
+    private static boolean isCurrentCommonField(final String key) {
+        return Arrays.asList(currentRequestCommonFields).contains(key);
+    }
+
+    private static boolean isLastCommonField(final String key) {
+        return Arrays.asList(lastRequestCommonFields).contains(key);
+    }
+
+    private static boolean isCurrentPlatformField(final String key) {
+        return Arrays.asList(currentRequestPlatformFields).contains(key);
+    }
+
+    private static boolean isLastPlatformField(final String key) {
+        return Arrays.asList(lastRequestPlatformFields).contains(key);
+    }
+
+    private static String[] getCurrentRequestCommonFields() {
+        return currentRequestCommonFields;
+    }
+
+    private static String[] getCurrentRequestPlatformFields() {
+        return currentRequestPlatformFields;
+    }
+
+    private static String[] getLastRequestCommonFields() {
+        return lastRequestCommonFields;
+    }
+
+    private static String[] getLastRequestPlatformFields() {
+        return lastRequestPlatformFields;
+    }
+
+    static String[] getCommonFields(final boolean isCurrent) {
```

Comment:
I suggest javadoc'ing `getCommonFields` and `getPlatformFields` to call out what the difference is.

(For those who don't have the spec committed to memory 😉 )

Reply:
Addressed